application/test/process/script_launcher_test.rb

```diff
@@ -185,11 +200,14 @@ class ScriptLauncherTest < ActiveSupport::TestCase
     @launcher.stubs(:to_time).with('2025-08-28T14:30:45').returns(Time.parse('2025-08-28T14:30:45'))
     @launcher.stubs(:elapsed).with(Time.parse('2025-08-28T14:30:45')).returns(120)
 
+    expected_log_filename = 'launch_detached_process-35-W2025.log'
```

Comment:
I would use the right file name `launch_detached_process-2025-W35.log` for these tests too to avoid confusion

Reply:
Fixed